<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>65</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>65</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>88</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа №8</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Введите время : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="time_line">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>hh:mm:ss</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_time">
        <property name="text">
         <string>Заполнить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Секунд до конца суток: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="res">
        <property name="text">
         <string>-----</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_go">
        <property name="text">
         <string>Сосчитать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
